src/code-challenges/more-object-keys.js/more-object-keys.js

```diff
@@ -0,0 +1,23 @@
+export function getHouses(arr){
+  return arr.map(element => Object.values(element)[3]);
+}
+
+export function updateNumbers(obj){
+  const array = (Object.entries(obj));
+
+  return array.map(arr => arr[0] + ': ' + arr[1]);
+}
+
+// export function houseSize(arr){
+//   const array = arr.array.forEach(element => {
+      
+//   });(element => Object.values(element));
+
+//   console.log(array);
+// }
+
+export function hasChildrenEntries(arr, character) {
+  const children = arr.map(person => person.children);
```

Comment:
I think what a you want is a separate things ex let children = false, then loop (forEach or map) and change that value based on conditions that you've looped through.